docs/getting-started/advanced-setting.asciidoc

```diff
@@ -67,6 +67,18 @@ IMPORTANT: {es-sec} requires {ecs-ref}[ECS-compliant data]. If you use third-par
 collectors to ship data to {es}, the data must be mapped to ECS.
 <<siem-field-reference>> lists ECS fields used in {es-sec}.
 
+[discrete]
+[[update-threat-intel-indices]]
+== Update default Elastic Security threat intelligence indices
+
+The `securitySolution:defaultThreatIndex` advanced setting defines threat intelligence indices that {elastic-sec} will use when collecting threat match data. One or more threat intelligence indices can be defined; the `filebeat-*` index is specified by default.
+
+The `securitySolution:defaultThreatIndex` setting controls features that query threat indices, such as the Threat Intelligence view on the Overview page and threat indicator match rules. You can enable Cross Cluster Search (CSS) by adding threat indices from remote clusters.
+
+NOTE: Indicator match rules typically run more queries than other rule types, which may affect rule performance.
+
+IMPORTANT: Threat intelligence indices aren't required to be ECS compatible when used in an indicator match rule. However, we recommend compatibility if you'd like for your alerts to be supplemented with relevant threat indicator information. By default, indicator match rules are configured to ingest indicators from the Threat Intel {filebeat} module and to check for fields prefixed with `threatintel.indicator`. This rule setting is stored in the *Indicator prefix override* field, under indicator match rule advanced settings. If your indicator data contains indicator fields at a different location, update this field accordingly to ensure alert enrichment can still be performed.
```

Comment:
```suggestion
IMPORTANT: Threat intelligence indices aren't required to be ECS compatible when used in an indicator match rule. However, we recommend compatibility if you'd like for your alerts to be enriched with relevant threat indicator information. By default, indicator match rules are configured to ingest indicators from the Threat Intel {filebeat} module and to check for fields prefixed with `threatintel.indicator`. This rule setting is stored in the *Indicator prefix override* field, under indicator match rule advanced settings. If your indicator data contains indicator fields at a different location, update this field accordingly to ensure proper alert enrichment.
```

I would use the verb "enrichment" here instead of "supplemented" as enrichment has a very specific meaning here. It might also be worth linking to the ECS docs here: https://www.elastic.co/guide/en/ecs/current/ecs-threat.html#field-threat-indicator-confidence